Jenkinsfile

```diff
@@ -181,6 +181,8 @@ spec:
            description: 'List of standalone TCK bundle file names to be appended to the base url' )
     choice(name: 'PROFILE', choices: 'FULL\nWEB', 
            description: 'Profile to be used for running CTS either web/full' )
+    choice(name: 'JDK', choices: 'JDK8\nJDK11',
```

Comment:
Looks good, so I think that the first listed option is taken as a default, so I think we will default to JDK8 if the parameter is not specified.

Reply:
Yes the default value for variable "JDK" will be JDK8 (the first option) .  Also by default JAVA_HOME will be JDK8 as it is set in the docker image.